diagnostic.md

```diff
@@ -20,6 +26,8 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
+After it executes the value of c is 7.
+The operators are c and c+d and they are used to assign a value to c
```

Comment:
The operators in this expression are `=` and `+`. `=` assigns value to whatever that's on the left of it, and `+` is addition.